fs/fs.go

```diff
@@ -41,6 +42,11 @@ import (
 
 var partitionRegex = regexp.MustCompile("^(:?(:?s|xv)d[a-z]+\\d*|dm-\\d+)$")
 
+const (
+	systemRoot   = "root"
```

Comment:
TODO to make these public and part of the API so people can use them

can we also append "Label" to them?